Domain/DTOs/OrderCreationDto.cs

```diff
@@ -2,5 +2,20 @@
 
 public class OrderCreationDto
 {
-    
+    public int TotalPrice { get; }
+    public PaymentMethod PaymentMethod { get;}
+    public DateTime CreatedAt { get;}
+    public PaymentStage PaymentStage { get;}
+    public int OwnerId { get; }
+    public int AddressId { get; }
+
+    public OrderCreationDto(int totalPrice, PaymentMethod paymentMethod, DateTime createdAt, PaymentStage paymentStage, int ownerId, int addressId)
```

Comment:
probably here instead of passing OwnerId and addressId, we will just pass all the properties needed for owner (name, email, phone) and address (street, city, country) and then in the logic the address will be connected with the customer.